Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -1927,6 +1927,55 @@ void ADefiWalletCoreActor::Erc1155Approve(FString contractAddress,
     });
 }
 
+void ADefiWalletCoreActor::SendDynamicContract(
+    FString contractAddress, int32 walletindex, FString abiJson,
+    FString functionName, FString functionArgs,
+    FDynamicContractSendDelegate Out) {
+
```

Comment:
should these all be under defiwalletcoreactor? or could there be a "contract actor" or something like that that one will initialize once with the common params (contract address, abijson, rpc address) and then just call a function on it?

Reply:
yes, that's possible. i'll refactor